docs/cedarling/uniffi/cedarling-kotlin.md

```diff
@@ -59,21 +82,23 @@ Please refer to [this document](./cedarling-uniffi.md) for details on the struct
 
 ### **Recipe 2:** Sample Java Maven project using the Kotlin binding
 
-1. Build Cedarling:
+1. Build Cedarling by executing below command from `./jans/jans-cedarling` of cloned jans project:
     ```bash
     cargo build -r -p cedarling_uniffi
     ```
    In `target/release`, you should find the `libcedarling_uniffi.dylib` (if Mac OS), `libcedarling_uniffi.so` (if Linux OS), or `libcedarling_uniffi.dll` (if Windows OS) file, depending on the operating system you are using.
+   **Note:** You can use pre-built `libcedarling_uniffi.so` from the [Jans releases page](https://github.com/JanssenProject/jans/releases). Ref: `https://github.com/JanssenProject/jans/releases/download/{version}/libcedarling_uniffi-{version}.so`
```

Comment:
There is lot of repetition between these notes and the `Building from Source` part. I think you can just tell the user to refer to that section and then follow steps.

Reply:
Done